src/proctored-exam-settings/ProctoredExamSettings.jsx

```diff
@@ -189,7 +194,7 @@ function ExamSettings(props) {
 
   function renderContent() {
     return (
-      <Form>
+      <Form onSubmit={handleSubmit} data-test-id="proctoringForm">
```

Comment:
Why can't you do `onSubmit={onButtonClick}` directly?
If needed, we should just update the function `onButtonClick`